packages/datadog-plugin-graphql/test/index.spec.js

```diff
@@ -167,7 +168,7 @@ describe('Plugin', () => {
       })
 
       describe('graphql-yoga', () => {
-        withVersions('graphql', 'graphql-yoga', version => {
+        withVersions(plugin, 'graphql-yoga', version => {
```

Comment:
I grepped the codebase and it looks like we only follow this pattern in a single other location, namely express. Should we be following this `withVersions(plugin` pattern everywhere?

Reply:
The issue arises when there are nested test cases using multiple instances of `withVersion`. With nested test cases, the reporting ends up taking the last dependency instead of the root plugin value being tested since previously they shared the same `graphql` key. To avoid that error, we give the dependencies a different key such that the root plugin version info is correctly used.

I looked at the codebase and it seems that `graphql` and `express` are the only two instances where there are nested cases of `withVersions` and both are now reporting correctly. There is no need to extend this to the rest of the repo since no other plugin has the nested use case